mixer/pkg/attribute/envoyProtoBag.go

```diff
@@ -168,6 +168,9 @@ func AccessLogProtoBag(msg *accesslog.StreamAccessLogsMessage, num int) *EnvoyPr
 
 	reqMap["context.proxy_error_code"] = ParseEnvoyResponseFlags(commonproperties.GetResponseFlags())
 
+	//for identifying the log type as Service Access Logs instead of Mixer Report
```

Comment:
Can you add some comment here about the value of this field on migration?

Reply:
Done